applications/jonny5/src/j5_authz_req.erl

```diff
@@ -16,154 +16,25 @@
-                       ],
-            send_response(lists:foldl(fun(F, R) -> F(R) end, Request, Routines));
-        {'error', _R} ->
-            lager:debug("unable to determine account id for ~s: ~p"
-                       ,[Number, _R]
-                       ),
-            'ok'
-    end.
-
--spec maybe_local_resource(knm_number_options:extra_options(), j5_request:request()) -> 'ok'.
-maybe_local_resource( Props, Request) ->
-    case knm_number_options:is_local_number(Props) of
-        'true' -> maybe_authz_local_resource(Request);
-        'false' ->
-            maybe_account_limited(Request)
-    end.
-
--spec maybe_authz_local_resource(j5_request:request()) -> 'ok'.
-maybe_authz_local_resource(Request) ->
-    case should_authz_local(Request) of
-        'false' -> allow_local_resource(Request);
-        'true' ->
-            lager:debug("authz_local_resources enabled, applying limits for local numbers"),
-            maybe_account_limited(Request)
-    end.
-
--spec allow_local_resource(j5_request:request()) -> 'ok'.
-allow_local_resource(Request) ->
-    Number = j5_request:number(Request),
-    lager:debug("number ~s is a local number for account ~s, allowing"
-               ,[Number, j5_request:account_id(Request)]
-               ),
-    Routines = [fun(R) -> j5_request:authorize_account(<<"limits_disabled">>, R) end
-               ,fun(R) -> j5_request:authorize_reseller(<<"limits_disabled">>, R) end
-               ],
-    send_response(lists:foldl(fun(F, R) -> F(R) end, Request, Routines)).
-
--spec should_authz_local(j5_request:request()) -> boolean().
-should_authz_local(Request) ->
-    Node = j5_request:node(Request),
-    kapps_config:get_is_true(<<"ecallmgr">>, <<"authz_local_resources">>, 'false', Node).
+    'true' = kapi_authz:authz_req_v(JObj),
+    Request = j5_request:from_jobj(JObj),
+    maybe_account_limited(Request).
 
 -spec maybe_account_limited(j5_request:request()) -> 'ok'.
 maybe_account_limited(Request) ->
     AccountId = j5_request:account_id(Request),
     Limits = j5_limits:get(AccountId),
     R = maybe_authorize(Request, Limits),
     case j5_request:is_authorized(R, Limits) of
-        'true' -> maybe_determine_reseller_id(R);
+        'true' -> maybe_reseller_limited(R);
         'false' ->
             lager:debug("account ~s is not authorized to create this channel"
                        ,[AccountId]
                        ),
             send_response(R)
     end.
 
--spec maybe_determine_reseller_id(j5_request:request()) -> 'ok'.
-maybe_determine_reseller_id(Request) ->
-    case j5_request:reseller_id(Request) of
-        'undefined' -> determine_reseller_id(Request);
-        _Else -> maybe_reseller_limited(Request)
-    end.
-
--spec determine_reseller_id(j5_request:request()) -> 'ok'.
-determine_reseller_id(Request) ->
-    AccountId = j5_request:account_id(Request),
-    ResellerId = kz_services_reseller:get_id(AccountId),
```

Comment:
This is the correct way to get reseller id, `kzd_accounts:reseller_id/1` is just looking at the the account's doc.